resource_config.yaml

```diff
@@ -194,7 +179,16 @@ resources:
       - create
       - delete
       - error
-
+  # - name: litmuschaos.io/v1alpha1/chaosengines
+  #   namespaces:
+  #     include:
+  #       - all
+  #     ignore:
+  #       -
+  #   events:
+  #     - create
+  #     - delete
+  #     - error
```

Comment:
```suggestion
```

Reply:
cc: @PrasadG193 , I have had tried with a CR it's working fine.

Reply:
Please remove commented lines